pkg/config/config_template.yaml

```diff
@@ -1180,6 +1180,11 @@ api_key:
 #
 # jmx_use_container_support: false
 
+## @param jmx_log_file - string - optional
```

Comment:
What do you think about adding the default value, ie `- default: jmxfetch.log`?

Reply:
The default depends on the OS and should be an absolute path, so it's tricky to document it here (and should not be documented as a relative path). The `log_file` config option links to this [docs page](https://docs.datadoghq.com/agent/guide/agent-log-files/?tab=agentv6v7) that we should update for 7.24 (in a future PR) to mention this new file.